matplotlibrc.template

```diff
@@ -433,6 +433,12 @@
 #date.autoformatter.second:      %H:%M:%S
 #date.autoformatter.microsecond: %M:%S.%f
 
+## This sets the epoch that matplotlib uses to convert dates to floats.
+## This date is day 0.0, and matplotlib converts to floating days from
+## this date using `matplotlib.dates.date2num`.  This can also be
+## set using `matplotlib.dates.set_epoch`, but to prevent confusion, can only
+## be done so before the epoch is used for the first time.
```

Comment:
```suggestion
## The reference date for matplotlibs internal date representation
## See https://matplotlib.org/examples/ticks_and_spines/date_precision_and_epochs.py
```

It's better to link to relevant documentation instead of adding description here: No duplication/synchronization issues and the rc file stays more compact.